openshift/dynamic/discovery.py

```diff
@@ -1,22 +1,31 @@
 import os
 import six
-import json
 import hashlib
 import tempfile
 from collections import defaultdict
-from abc import abstractmethod, abstractproperty
 
 from urllib3.exceptions import ProtocolError, MaxRetryError
 
-from openshift import __version__
-from .exceptions import ResourceNotFoundError, ResourceNotUniqueError, ApiException, ServiceUnavailableError
-from .resource import Resource, ResourceList
-
+from kubernetes.dynamic.discovery import Discoverer as K8sDiscoverer
+from kubernetes.dynamic.discovery import LazyDiscoverer as K8sLazyDiscoverer
+from kubernetes.dynamic.discovery import LazyDiscoverer as K8sEagerDiscoverer
+from kubernetes.dynamic.discovery import (  # noqa
+    CacheEncoder,
+    CacheDecoder,
+    DISCOVERY_PREFIX,
+    ResourceGroup,
+)
+from kubernetes.dynamic.exceptions import (
+    ResourceNotFoundError,
+    ResourceNotUniqueError,
+    ApiException,
```

Comment:
Optional Nit: For pep8 these should be in alphabetical order, but not a big deal. If the need to be imported in this order, a comment would be nice.